config/flags.go

```diff
@@ -318,8 +318,6 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Int(SnowOptimalProcessingKey, 10, "Optimal number of processing containers in consensus")
 	fs.Int(SnowMaxProcessingKey, 256, "Maximum number of processing items to be considered healthy")
 	fs.Duration(SnowMaxTimeProcessingKey, 30*time.Second, "Maximum amount of time an item should be processing and still be healthy")
-	fs.Uint(SnowMixedQueryNumPushVdrKey, 10, fmt.Sprintf("If this node is a validator, when a container is inserted into consensus, send a Push Query to %s validators and a Pull Query to the others. Must be <= k.", SnowMixedQueryNumPushVdrKey))
```

Comment:
should we deprecate these first? removing flags breaks upgrades where flags are hard coded. we should probably consider a general policy here? curious your thoughts @StephenButtolph

Reply:
I think its reasonable to not remove flags on a patch version in general.

Reply:
The way we've historically done this is based on the perceived popularity of the flags... I don't think anyone has ever used these.

But having a general `deprecate -> remove` flow isn't unreasonable.

Reply:
going to punt this as a followup issue to clarify our stance here (but not change this PR)